docs/data/date-pickers/fields/fields.md

```diff
@@ -37,25 +37,29 @@ The single input range fields are not supported on the range pickers yet (you ca
 
 The field components have an internal state to update the visible value.
 
-It will only call the `onChange` callback when all the sections of the modified date are filled.
+It will only call the `onChange` callback when:
 
-#### On single element fields
+- all the sections of the modified date are filled
+- all the sections of the modified date are empty (will return `null` for the modified date)
+- all the sections of the modified date were filled, and the user cleaned the content of one section (will return `Invalid date` for modified date)
 
-On a single date field (`DateField` / `DateTimeField` / `TimeField`),
-`onChange` will be called if all the sections are filled.
+#### On simple fields
```

Comment:
I think we don't need a specific section for "simple fields"; after the condition list, we can jump to the example, and it's introduction directly.

```suggestion
```